### ISSUE TITLE
Invalid URL generated in WSDL for included XSD files

### ISSUE THREAD

**Issue by seckinsq** [quarkiverse/quarkus-cxf#345]
We are having a similar issue as #323 but we are using our own WSDL file and we use some include and imports in WSDL or XSD files. When we want to retrieve wsdl from service (?wsdl), the URL used for _schemaLocation_ on _xs:include_ is not valid.

`<xs:include schemaLocation="http://localhost:8080/soap/CxftestSoapService?wsdl?xsd=cxftest.xsd"/>`

It seems like application is adding path query for XSD to WSDL url instead of service URL. 

http://localhost:8080/soap/CxftestSoapService?wsdl?xsd=cxftest.xsd 
(Gives soap:Server No binding operation info while invoking unknown method with params unknown.)

vs 

http://localhost:8080/soap/CxftestSoapService?xsd=cxftest.xsd 
(This works but if it imports another XSD, its query path is added to this XSD's url  too : CxftestSoapService?xsd=cxftest.xsd?xsd=common.xsd)

Example mini-project to reproduce:
[code-with-quarkus.zip](https://github.com/quarkiverse/quarkus-cxf/files/7906988/code-with-quarkus.zip)

**Comment by shumonsharif** [comment]
Hi @seckinsq Thanks for the issue.  Did you by any chance try exploring the `published-endpoint-url` property as a workaround?  I believe you should be able to set that value to either an _absolute_ or _relative_ URL ... can you please try and let us know your findings?

**Comment by seckinsq** [comment]
Hi, @shumonsharif thank you for fast response. 

We tried absolute URL and it was OK but we cannot use it as there are many environments where this application will be running. All will have different host (and there may be different versions)

I have tried your suggestion with relative URL : 
`quarkus.cxf.endpoint."CxftestSoapService".published-endpoint-url=/soap/CxftestSoapService` 

For http://localhost:8080/soap/CxftestSoapService?wsdl
`<xs:include schemaLocation="/soap/CxftestSoapService?xsd=cxftest.xsd"/>`

For http://localhost:8080/soap/CxftestSoapService?xsd=cxftest.xsd
`<xs:import schemaLocation="/soap/CxftestSoapService?xsd=common.xsd" namespace="http://cxftest.com/common/schema"/>`

So they are looking good with relative URL, I will check with client applications if this can be acceptable.
Thanks.
